src/game.cpp

```diff
@@ -798,6 +798,7 @@ bool game::start_game()
     load_map( lev, /*pump_events=*/true );
 
     int level = m.get_abs_sub().z;
+    u.setpos( tripoint_south );
```

Comment:
As start_position::place_player() re-invokes cache builds anyway, it should be fine to place the player at an arbitrary position on the map, which tripoint_south does, but just to more closely comply with game invariants, I'd rather stick the player in the central submap instead of in a corner.
```suggestion
    u.setpos( project_to<coords::ms>( omtstart );  );
```